src/NServiceBus.Transport.Msmq/Scripts/NServiceBus.Transport.Msmq.targets

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)\Scripts\*.ps1">
+      <Link>Scripts\%(RecursiveDir)%(Filename)%(Extension)</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
If this package is used with an SDK-style project, these scripts are going to show up in Solution Explorer, but they won't for either of the old csproj combos (packages.config or package references)

I recommend adding `<Visible>false</Visible>` so that they are hidden everywhere, especially since if someone does try to edit it, they will be modifying the version in the global NuGet package folder.

Reply:
Nice! I didn't know that existed. Not showing is better, totally agree.

Reply:
Add visibility false and the copy magic to avoid having the linked files/folder show up in solution explorer.